poleno/utils/tests/test_templatetags.py

```diff
@@ -253,3 +256,9 @@ def test_change_lang_tag(self):
             self.assertEqual(r1.content, u'(/en/language/)(/de/language/)(/fr/language/)')
             self.assertEqual(r2.content, u'(/en/language/)(/de/language/)(/fr/language/)')
             self.assertEqual(r3.content, u'(/en/language/)(/de/language/)(/fr/language/)')
+
+        patched = functools.partial(page_not_found, template_name=u'404x.html')
+        with mock.patch(u'django.views.defaults.page_not_found', patched):
```

Comment:
@viliambalaz Pls oddel to do samostatneho testu. Nieco ako `test_change_lang_tag_with_missing_language_in_url`

Reply:
:ok: